coriolis/tests/api/v1/test_regions.py

```diff
@@ -0,0 +1,241 @@
+        mock_get_region.assert_called_once_with(mock_context, id)
+
+    @mock.patch.object(region_view, 'collection')
+    @mock.patch.object(api.API, 'get_regions')
+    def test_index(
+        self,
+        mock_get_regions,
+        mock_collection
+    ):
+        mock_req = mock.Mock()
+        mock_context = mock.Mock()
+        mock_req.environ = {'coriolis.context': mock_context}
+
+        result = self.regions.index(mock_req)
+
+        self.assertEqual(
+            mock_collection.return_value,
+            result
+        )
+
+        mock_context.can.assert_called_once_with("migration:regions:list")
+        mock_get_regions.assert_called_once_with(mock_context)
+        mock_collection.assert_called_once_with(mock_get_regions.return_value)
+
+    def test_validate_create_body(
+        self,
+    ):
+        name = mock.sentinel.name
+        description = mock.sentinel.description
+        enabled = False
+        body = {
+            "region": {
+                "name": name,
+                "description": description,
+                "enabled": enabled
+            }
+        }
+
+        result = testutils.get_wrapped_function(
+            self.regions._validate_create_body)(self.regions, body=body)
+
+        self.assertEqual(
+            (name, description, enabled),
+            result
+        )
+
+    @mock.patch.object(region_view, 'single')
+    @mock.patch.object(api.API, 'create')
+    @mock.patch.object(regions.RegionController, '_validate_create_body')
+    def test_create(
+        self,
+        mock_validate_create_body,
+        mock_create,
+        mock_single
+    ):
+        mock_req = mock.Mock()
+        mock_context = mock.Mock()
+        mock_req.environ = {'coriolis.context': mock_context}
+        name = mock.sentinel.name
+        description = mock.sentinel.description
+        enabled = True
+        mock_validate_create_body.return_value = (name, description, enabled)
+        region = {
+            "name": name,
+            "description": description,
+            "enabled": enabled
+        }
+        body = {"region": region}
+
+        result = self.regions.create(mock_req, body)
+
+        self.assertEqual(
+            mock_single.return_value,
+            result
+        )
+
+        mock_context.can.assert_called_once_with("migration:regions:create")
+        mock_validate_create_body.assert_called_once_with(body)
+        mock_create.assert_called_once_with(
+            mock_context, region_name=name, description=description,
+            enabled=enabled)
+        mock_single.assert_called_once_with(mock_create.return_value)
+
+    def test_validate_update_body(
+        self,
+    ):
+        name = mock.sentinel.name
+        description = mock.sentinel.description
+        enabled = True
+        region = {
+            "name": name,
+            "description": description,
+            "enabled": enabled
```

Comment:
Please add an additional key: value in this `region` map. The expected result should be the original map that contains only these 3 key-values. That's the point of the method, so you should test that also